src/Controller/Component/ModulesComponent.php

```diff
@@ -670,6 +670,9 @@ protected function folderChildrenRelated(string $id, array $relatedIds)
             }
             // invert relation call => use 'parent' relation on children folder
             $data = compact('id') + ['type' => 'folders'];
+            if (Hash::check($item, 'meta')) {
+                $data += ['meta' => Hash::get($item, 'meta')];
```

Comment:
to make scrutinizer happy
```suggestion
                $data += ['meta' => (array)Hash::get($item, 'meta')];
```